swift-ticketing/test/swift_ticketing/fixtures.clj

```diff
@@ -1,20 +1,53 @@
 (ns swift-ticketing.fixtures
   (:require
-   [next.jdbc :as jdbc]
-   [honey.sql :as sql]
-   [swift-ticketing.core :refer [create-connection-pool]]
+   [com.stuartsierra.component :as component]
+   [swift-ticketing.core :refer [swift-ticketing-system]]
    [swift-ticketing.factory :as factory]
    [swift-ticketing.config :refer [read-test-config]]
-   [swift-ticketing.migrations :as migrations]))
+   [swift-ticketing.migrations :as migrations]
+   [next.jdbc :as jdbc]
+   [honey.sql :as sql]))
 
 (def test-env
-  (let [db-config (:database (read-test-config))]
-    {:db-spec (create-connection-pool db-config)}))
+  {:test-user-id nil
+   :db-spec nil})
+
+(def swift-ticketing-test-system
+  (swift-ticketing-system (read-test-config)))
+
+(defn start-test-system []
+  (alter-var-root #'swift-ticketing-test-system component/start))
 
-(defn run-migrations [tests]
-  (let [db-config (:database (read-test-config))]
+(defn stop-test-system []
+  (alter-var-root #'swift-ticketing-test-system component/stop))
+
+(defn setup-test-system [tests]
```

Comment:
This sets up a LOT of things. Not sure this should be called via a one-time fixture from handlers-test, but rather something more closer to running tests. Something like Goose's [test-runner](https://github.com/nilenso/goose/blob/main/test/test_runner.clj) maybe?

Reply:
Reason for pulling this part above is so when you have other tests that require DB setup, you don't have to do that via fixtures.

Reply:
Added [kaocha](https://github.com/lambdaisland/kaocha) as the test runner